app/controllers/SmallProducerDetailsController.scala

```diff
@@ -18,57 +18,56 @@ package controllers
 
 import controllers.actions._
 import forms.SmallProducerDetailsFormProvider
-
-import javax.inject.Inject
-import models.{Mode, SmallProducer, UserAnswers}
+import viewmodels.checkAnswers.SmallProducerDetailsSummary
+import views.html.SmallProducerDetailsView
+import models.{Mode, SmallProducer}
 import navigation.Navigator
 import pages.SmallProducerDetailsPage
 import play.api.i18n.{I18nSupport, MessagesApi}
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import repositories.SessionRepository
 import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.{SummaryList, SummaryListRow}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
-import viewmodels.checkAnswers.SmallProducerDetailsSummary
-import views.html.SmallProducerDetailsView
 import viewmodels.govuk.summarylist._
 
-
+import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
 class SmallProducerDetailsController @Inject()(
-                                         override val messagesApi: MessagesApi,
-                                         sessionRepository: SessionRepository,
-                                         navigator: Navigator,
-                                         identify: IdentifierAction,
-                                         getData: DataRetrievalAction,
-                                         requireData: DataRequiredAction,
-                                         formProvider: SmallProducerDetailsFormProvider,
-                                         val controllerComponents: MessagesControllerComponents,
-                                         view: SmallProducerDetailsView
-                                 )(implicit ec: ExecutionContext) extends FrontendBaseController with I18nSupport {
+                                                override val messagesApi: MessagesApi,
+                                                sessionRepository: SessionRepository,
+                                                navigator: Navigator,
+                                                identify: IdentifierAction,
+                                                getData: DataRetrievalAction,
+                                                requireData: DataRequiredAction,
+                                                formProvider: SmallProducerDetailsFormProvider,
+                                                val controllerComponents: MessagesControllerComponents,
+                                                view: SmallProducerDetailsView
+                                              )(implicit ec: ExecutionContext) extends FrontendBaseController with I18nSupport {
 
   val form = formProvider()
 
   val spList = List(SmallProducer("ABC Ltd", "SDIL123456", (1000L, 1000L)),
```

Comment:
now this data should come from the previous page and this hard coded values should be removed